CHANGELOG.md

```diff
@@ -15,12 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `hypothesis` strategies and roundtrip test for kernels, constraints and objectives
 
 ### Changed
-- `torch` numeric types are now loaded lazily
 - Reorganized acquisition.py into `acquisition` subpackage
-- `torch` is imported lazily in `surrogates`
 - Acquisition functions are now their own objects
 - `acquisition_function_cls` constructor parameter renamed to `acquisition_function`
+- `torch`, `gpytorch` and `botorch` are lazy-loaded for improved startup time
 
+### Removed
```

Comment:
@AdrianSosic will review after the preceding PR was merged. Would appreciate a quick ping once this is the case :)

Reply:
Sure, thanks. "Ping" btw :D